dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/entities/MessageEntity.kt

```diff
@@ -0,0 +1,63 @@
+package org.dataland.datalandcommunitymanager.entities
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.dataland.datalandcommunitymanager.model.dataRequest.StoredDataRequestMessageObject
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * The database entity for storing a single message in a data request conversation
+ */
+@Entity
+@Table(name = "messages")
+data class MessageEntity(
+    @Id
+    val messageId: String,
+
+    val ordinal: Int,
+
+    @Column(columnDefinition = "TEXT")
+    val contacts: String?,
+
+    @Column(columnDefinition = "TEXT")
+    val message: String?,
+
+    val lastModifiedDate: Long,
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "data_request_id")
+    var dataRequest: DataRequestEntity?,
```

Comment:
Wieso ist das nullable.  Eine message entity kann doch nur in Zusammenhang mit einer request geschrieben worden sein, oder?

Reply:
Das ist wahrscheinlich nicht notwendig so wie wir das cross referencen lösen. To be done.

Reply:
Habs entfernt. baut hoffentlich noch.